src/components/Bookstore.jsx

```diff
@@ -16,13 +17,13 @@ const Bookstore = () => {
     <>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230380577-b8237a27-4bee-4238-9584-8fa949570769.png)

- [ ] It would be nice if you make your Getting Started Section something like the below 👇 to make your readme descriptive and professional 👍

````md
## 💻 Getting Started <a name="getting-started"></a>

> How to get started and make use of this project.

To get a local copy up and running, follow these steps.

### Prerequisites

In order to run this project you need: 

```
Visual Studio Code
Node.js
```

### Setup

Clone this repository to your desired folder

```
cd your-folder
https://github.com/jlvFlores/Bookstore
cd Bookstore
```

### Install

Install all dependencies to this project with:

```
npm install
```

### Usage

To run the project, execute the following command:  

```
npm start
```

### Run tests

To run tests, run the following command: 

```
npm test
```

### Deployment

You can use any deployment service to deploy the application.

```
npm run build
```

<p align="right">(<a href="#readme-top">back to top</a>)</p>